client/src/sockets/receiveEvents.js

```diff
@@ -14,5 +19,9 @@ export const receiveEvents = (dispatch, socket) => {
     socket.on(event, data => dispatch({ type: event, payload: data }))
   );
 
+  Object.values(ListEvents).forEach(event =>
+    socket.on(event, data => dispatch({ type: event, payload: data }))
+  );
+
```

Comment:
Please create separate methods instead of comments:
`receiveItemEvents` and `receiveListEvents`